.ci.yaml

```diff
@@ -324,14 +324,22 @@ targets:
       jazzy_version: "0.14.1"
     timeout: 75
 
-  - name: Mac mac_android_aot_engine
+  # TODO(keyonghan): Temporarily use Linux VM to host Mac engine_v2 builders.
+  # To change it back to `Mac` when there is enough capacity.
+  # https://github.com/flutter/flutter/issues/114656
+  - name: Linux mac_android_aot_engine
```

Comment:
We may want to select the initial platform the build will be running on using something different than the name.

Reply:
This doesn't seem to work.
Here is an led run from a linux vm: https://luci-milo.appspot.com/raw/build/logs.chromium.org/flutter/led/keyonghan_google.com/3e1876d68d11f316d53063a79b86f480c9088371897948b3f47af03ac4efc78b/+/build.proto
(got configs based on `Linux linux_host_engine`, and then update `config` from `linux_host_engine` to `mac_host_engine`)

Reply:
This is because the osx_sdk is propagated from the top task to the subbuild. Can you please add that property to ci/builders/mac_ files?

Reply:
There doesn't seem a `drone_properties` in the .json files yet. I guess we need to support that in recipes first.

Reply:
Can you try properties?

Reply:
Kicked off another led run based on newly added properties: https://luci-milo.appspot.com/raw/build/logs.chromium.org/flutter/led/keyonghan_google.com/32febe6e7ee6fbfc0924be32e77993b4b6d6288a906420846ff2a2987afdf7fd/+/build.proto

Reply:
huh, it passed all the sub builds, but fails with a new error on step `Global generators.Release-FlutterMacOS.framework`:
```
Cannot find dsymutil at /b/s/w/ir/cache/builder/src/flutter/sky/tools/../../../buildtools/mac-x64/clang/bin/dsymutil
```
Do you have any insight?

Reply:
I guess the reason is we are running macos scripts on a linux vm.

Reply:
yes, the builds with global generators need to run mac orchestrators.

Reply:
Only `mac_android_aot_engine` doesn't have `global generators`, and restored other builders. 

Validation for `mac_android_aot_engine`: https://luci-milo.appspot.com/raw/build/logs.chromium.org/flutter/led/keyonghan_google.com/4d7203206d2a76005cd388a4389ab793f0c96821882303ed1f93f90a65f92bef/+/build.proto